pkg/services/queryhistory/database.go

```diff
@@ -25,8 +25,32 @@ func (s QueryHistoryService) createQuery(ctx context.Context, user *models.Signe
 		return err
 	})
 	if err != nil {
-		return err
+		return QueryHistoryDTO{}, err
+	}
+
+	dto := QueryHistoryDTO{
+		UID:           queryHistory.UID,
+		DatasourceUID: queryHistory.DatasourceUID,
+		CreatedBy:     queryHistory.CreatedBy,
+		CreatedAt:     queryHistory.CreatedAt,
+		Comment:       queryHistory.Comment,
+		Queries:       queryHistory.Queries,
+		Starred:       false,
 	}
 
-	return nil
+	return dto, nil
+}
+
+func (s QueryHistoryService) deleteQuery(ctx context.Context, user *models.SignedInUser, UID string) (int64, error) {
```

Comment:
the pr seems in shape already 🎉 , here I would put this function into sqlstore instead of leaving sql queries in the service pacakge.

Reply:
@ying-jeanne I got feedback previously to put everything under pkg/services/queryhistory same as it happens for [libraryelements](https://github.com/grafana/grafana/tree/main/pkg/services/libraryelements). The feedback is here: https://github.com/grafana/grafana/pull/44479#pullrequestreview-863516458

Reply:
@papagian Hey Sofia, is there any specific reason we want to leave the SQL query here? I thought we would like to keep everything interact with DB in the sqlstore service